utilities/discordutils.py

```diff
@@ -28,29 +28,34 @@ def parse_discord_roles(self, roles: list[discord.Role]) -> None:
 
 class DiscordChannel(ServiceChannel):
     def __init__(self, channel: discord.abc.Messageable, server: Optional[ServiceServer]):
-        self._channel = channel
+        self._channel: discord.abc.Messageable = channel
         self.history = channel.history
-        if not isinstance(channel, discord.DMChannel):
+        if not isinstance(channel, discord.abc.Messageable):
```

Comment:
@tayler6000  what is this about, please?  either `channel` parameter has some type and assignment on line 31 should infer the type and this condition on line 33 doesn't make any sense at all

..or the parameter type is incorrect and `channel.id` and `channel.name` are both unsafe access - `not isinstance(...)` will NOT make it safe - so why this particular condition?

Reply:
Looks like tayler is awol. What do you think resolution of this should be? Should we remove the isinstance checks for `channel` and just do:

```python
channel_id = channel.id
name = ""
```